web/tests/stream_data.test.js

```diff
@@ -820,7 +830,8 @@ test("create_sub", () => {
 
 test("creator_id", () => {
     people.add_active_user(test_user);
-
+    realm.realm_can_access_all_users_group = everyone_group.id;
```

Comment:
Did not mock the `settings_data` function here as there is one other function from that file which is used in the tests in this file and it would have needed `override_rewire` to override one function and not the other. So, instead just added a group so that `is_user_in_group` works as expected.